app/backend/graphql/user/mutations/userCapacityUpdateMutationField.test.ts

```diff
@@ -0,0 +1,80 @@
+import { PrismaClient } from '@prisma/client'
+import { GraphQLError } from 'graphql'
+import { getTestServer } from '../../../getTestServer'
+
+const prisma = new PrismaClient()
+
+const userCapacityUpdateMutation = `
+mutation userCapacityUpdate($userId: ID!, $capacityMinutes: Float!) {
+    userCapacityUpdate(userId: $userId, capacityMinutes: $capacityMinutes) {
+    id
+    capacityMinutes
+  }
+}
+`
+
+describe('userCapatityUpdate', () => {
+  beforeEach(async () => {
+    await prisma.user.deleteMany()
+    await prisma.team.deleteMany()
+    await prisma.teamMembership.deleteMany()
+  })
+
+  it('should be error when update capacity hours', async () => {
```

Comment:
Maybe use a bit more precise test description.

Something like that:
```suggestion
  it('should throw an error when the user is not an admin', async () => {
```

Reply:
@WINt1703 can you change this as Pascal suggested.